tools/ci_build/verify/check_typing_info.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+from types import ModuleType
+import inspect
+
+import tensorflow_addons
+
+EXAMPLE_URL = 'https://github.com/tensorflow/addons/blob/fa8e966d987fd9b0d20551a666e44e2790fdf9c7/tensorflow_addons/layers/normalizations.py#L73'
+TUTORIAL_URL = 'https://docs.python.org/3/library/typing.html'
+
+
+# TODO: add types and remove all elements from
+# the exception list.
+EXCEPTION_LIST = [
```

Comment:
I like this system as an iterative way to add typing :) 

Probably worth describing this system in the docs?

Reply:
Done